rythmbadge/request_handler_lib.c

```diff
@@ -431,13 +431,10 @@ static void status_response_handler(void * p_event_data, uint16_t event_size)
 		return;
 
 	response_event.response.which_type = Response_status_response_tag;
-	response_event.response.type.status_response.clock_status = advertiser_get_status_flag_is_clock_synced();
-	response_event.response.type.status_response.microphone_status = advertiser_get_status_flag_microphone_enabled();
-	response_event.response.type.status_response.scan_status = advertiser_get_status_flag_scan_enabled();
-	response_event.response.type.status_response.imu_status = advertiser_get_status_flag_imu_enabled();
-	response_event.response.type.status_response.battery_level = get_battery_level();
-	response_event.response.type.status_response.pdm_data = pdm_buf[0].mic_buf[0];
-	response_event.response.type.status_response.scan_data = ble_get_scan_rssi();
```

Comment:
Looking at the diff, I think you are deleting too many lines here. I think the correct way to merge this is to add the changes that you are adding while keeping as many of the previous lines as possible but please double check this.